src/Workspaces/Core/Portable/Editing/ImportAdderService.cs

```diff
@@ -234,7 +234,7 @@ CancellationToken cancellationToken
                     foreach (var namedType in SymbolAnnotation.GetSymbols(annotation, model.Compilation).OfType<INamedTypeSymbol>())
                     {
                         cancellationToken.ThrowIfCancellationRequested();
-                        if (namedType.OriginalDefinition.IsSpecialType() || namedType.IsNullable())
+                        if (namedType.OriginalDefinition.IsSpecialType() || namedType.IsNullable() || namedType.IsTupleType)
```

Comment:
why was this needed?

Reply:
From memory, without this a bunch of tests started adding `using System;` where it wasn't needed (tuple syntax doesn't need such an import).